### PR TITLE
mapproxy 1.14.0 (new formula)

### PR DESCRIPTION
**Pull request by singingwolfboy**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

New formula for [MapProxy](https://mapproxy.org)

### CODE REVIEW

File: Formula/mapproxy.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+  desc "Accelerating web map proxy"
+  homepage "https://mapproxy.org/"
+  url "https://files.pythonhosted.org/packages/98/3b/424fecff5f852100a100e326cc554df17da59d491d3c62fd305caceffcf8/MapProxy-1.14.0.tar.gz"
+  sha256 "dd36278d60cdcaaf31f7f9bbc50e90e770f3feb65cf4b3eff287215ee85f018d"
+  license "Apache-2.0"
+
+  depends_on "pillow"
+  depends_on "proj"
+  depends_on "python@3.9"
```
Comment (carlocab) [inline Formula/mapproxy.rb:12]: Can we use `python@3.10`?

File: Formula/mapproxy.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    sha256 "68fb519c14306fec9720a2a5b45bc9f0c8d1b9c72adf45c37baedfcd949c35a2"
+  end
+
+  def install
+    virtualenv_install_with_resources
+  end
+
+  test do
+    system bin/"mapproxy-util", "create", "-t", "base-config", testpath
+    assert_match("seed.yaml", shell_output("ls #{testpath}"))
```
Comment (carlocab) [inline Formula/mapproxy.rb:35]:
```suggestion
    assert_predicate testpath/"seed.yaml", :exist?
```

**Review by carlocab — COMMENTED** [review]
Thanks, @singingwolfboy. Just a few comments.

File: Formula/mapproxy.rb (hunk 1 above)
Comment (singingwolfboy) [reply Formula/mapproxy.rb:12]: I'd like to, but there was a problem with conflicting versions of Python, according to CI. See the first commit of this pull request for the specific error message.

**Review by carlocab — APPROVED** [review]
Thanks!

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**